quic/s2n-quic-core/src/varint/tests.rs

```diff
@@ -68,3 +68,87 @@ sequence_test!(four_byte_sequence_test(
 sequence_test!(two_byte_sequence_test([0x7b, 0xbd], 15293));
 
 sequence_test!(one_byte_sequence_test([0x25], 37));
+
+mod tests {
+    use super::*;
+    use s2n_codec::assert_codec_round_trip_value;
+
+    #[cfg(kani)]
+    #[kani::proof]
```

Comment:
For trivial harnesses like these, can you implement bolero harnesses for these instead? My complaint about `#[cfg(kani)]` is:
* we don't get any type checking or analysis with a regular `cargo build`
* we won't find out this is broken until `cargo kani` is executed, which could be trying to debug in CI

Reply:
Sure, lemme switch to bolero